File: W4Day1/Python/IfElse1/ThreeSiblings.py
sibling1 = input()
sibling2 = input()
sibling3 = input()

age1 = int(input())
age2 = int(input())
age3 = int(input())

if(age1 > age2) and (age1 > age3):
    if(age2 > age3):
        print(sibling1,"is the oldest,", sibling2,"is the middle,", sibling3,"is the youngest")
    else:
        print(sibling1,"is the oldest,", sibling3,"is the middle,", sibling2,"is the youngest")
elif(age2 > age1) and (age2 > age3):
    if(age1 > age3):
        print(sibling2,"is the oldest,", sibling1,"is the middle,", sibling3,"is the youngest")
    else:
        print(sibling2,"is the oldest,", sibling3,"is the middle,", sibling1,"is the youngest")
else:
    if(age3 > age1) and (age3 > age2):
        print(sibling3,"is the oldest,", sibling1,"is the middle,", sibling2,"is the youngest")
    else:
        print(sibling3,"is the oldest,", sibling2,"is the middle,", sibling1,"is the youngest")

File: W4Day1/IfElse1/ThreeSiblings.java
package W4Day1.IfElse1;

import java.util.*;

public class ThreeSiblings {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String sib1 = sc.nextLine();
        String sib2 = sc.nextLine();
        String sib3 = sc.nextLine();
        int age1 = sc.nextInt();
        int age2 = sc.nextInt();
        int age3 = sc.nextInt();

        sc.close();

        if (age1 >= age2 && age1 >= age3) {
            if (age2 >= age3) {
                System.out.printf("%s is the oldest, %s is the middle, and %s is the youngest\n", sib1, sib2, sib3);
            } else {
                System.out.printf("%s is the oldest, %s is the middle, and %s is the youngest\n", sib1, sib3, sib2);
            }
        } else if (age2 >= age1 && age2 >= age3) {
            if (age1 >= age3) {
                System.out.printf("%s is the oldest, %s is the middle, and %s is the youngest\n", sib2, sib1, sib3);
            } else {
                System.out.printf("%s is the oldest, %s is the middle, and %s is the youngest\n", sib2, sib3, sib1);
            }
        } else {
            if (age1 >= age2) {
                System.out.printf("%s is the oldest, %s is the middle, and %s is the youngest\n", sib3, sib1, sib2);
            } else {
                System.out.printf("%s is the oldest, %s is the middle, and %s is the youngest\n", sib3, sib2, sib1);
            }
        }
    }
}
